src/TesterInternal/GeoClusterTests/BasicMultiClusterTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Orleans;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using System.Net;
+using Orleans.TestingHost;
+
+
+// ReSharper disable InconsistentNaming
+
+namespace Tests.GeoClusterTests
```

Comment:
Oh, and I just remembered: it is using the TestHooks which is internal to the Orleans Runtime.